core/services/offchainreporting/peerstore.go

```diff
@@ -118,7 +118,7 @@ func (p *Pstorewrapper) readFromDB() error {
 }
 
 func (p *Pstorewrapper) getPeers() (peers []P2PPeer, err error) {
-	rows, err := p.db.DB().QueryContext(p.ctx, `SELECT id, addr FROM p2p_peers WHERE peer_id = $1`, p.peerID)
+	rows, err := p.db.Raw(`SELECT id, addr FROM p2p_peers WHERE peer_id = ?`, p.peerID).Rows()
```

Comment:
Ah. It doesn't use the context any more. Is there a `RawContext` or `WithContext` or similar?